kas/iot2050.yml

```diff
@@ -44,7 +44,7 @@ repos:
 
   cip-core:
     url: https://gitlab.com/cip-project/cip-core/isar-cip-core.git
-    refspec: 0feddaa6dd65c5a1b56ff6fcb18b4e748e53972f
+    refspec: ed318f247beef10e0433c6ed699b4c1ab008fa6a
```

Comment:
Has to come in lock-step with the isar update - like we did in isar-cip-core as well.